siliconcompiler/schema/schema_cfg.py

```diff
@@ -215,6 +215,74 @@ def schema_cfg():
     # Packaging
     cfg = schema_package(cfg)
 
+    # Packaging
+    cfg = schema_schematic(cfg)
+
+    return cfg
+
+
+###############################################################################
+# SCHEMATIC
+###############################################################################
+def schema_schematic(cfg):
+    ''' Schematic
+    '''
+
+    scparam(cfg, ['schematic', 'component'],
+            sctype='[(str,str)]',
+            shorthelp="Schematic: components",
+            switch="-schematic_component <(str,str)>",
+            example=["cli: -schematic_component (B0,NAND2X1)",
+                     "api: chip.set('schematic', 'component', ('B0','NAND2X1'))"],
+            schelp="""Components in the design, specified as a list of
+            (INSTANCE,MODEL) tuples.""")
```

Comment:
`schematic`, `component`, name, `model`, model? that is much more inline with the other schema keys

Reply:
Fixed and committed.